### PR TITLE
Enabled multi targeted nuget package build

### PR DESCRIPTION
**Pull request by daffers**
I needed to be able to target .net 4.0. I've added scripts to generate a multi targeted nuget package. Would you be able to upload this to nuget soon? 

I did notice that the existing nuget package indicates that the assembly is a net40 assembly while it actually targets net45.

### CODE REVIEW

File: buildandpack.cmd [hunk 1]
```diff
@@ -0,0 +1,2 @@
+msbuild .\src\WebAPI.Testing\WebAPI.Testing.csproj /t:Build /p:Configuration="Release 4.5"
```
Comment (jchannon) [inline buildandpack.cmd]: Should this have Package as well?

Comment (daffers) [inline buildandpack.cmd]: No, its only the final step that needs to package. The package step sweeps all the assemblies together. We could probably break out the package step into its own command call - i did just copy this method from here http://brian-federici.com/blog/2013/4/17/automatically-target-multiple-frameworks-in-your-nuget-package

**Comment by jchannon** [comment]
Thanks.

Out of interest are you using OWIN?

**Comment by daffers** [comment]
No we are not. I have used it in the past but we are a little stuck on net40. If you could let me know when the new package is up, it would be appreciated :)

**Comment by jchannon** [comment]
I've tried to do this but failing hard. I think its missing some nuget references maybe webapi? If I give yo commit rights and make you a co-owner on nuget would you be happy to update packages.config and nuget?

**Pull request merged**